doc/api/stream.md

```diff
@@ -1505,24 +1713,25 @@ var writable = new stream.Writable({
 
 <!--type=misc-->
 
-Both Writable and Readable streams will buffer data on an internal
-object which can be retrieved from `_writableState.getBuffer()` or
-`_readableState.buffer`, respectively.
+Both [Writable][] and [Readable][] streams will buffer data on an internal
+object which can be retrieved using `writable._writableState.getBuffer()` or
+`readable._readableState.buffer`, respectively.
 
-The amount of data that will potentially be buffered depends on the
-`highWaterMark` option which is passed into the constructor.
+The amount of data potentially be buffered depends on the `highWaterMark` option
+which is passed into the streams constructor.
```

Comment:
How does data contribute to the buffer size? Is it a count of chunks or the sum of the size of all chunks?